src/components/MainParticipantInfo/MainParticipantInfo.tsx

```diff
@@ -173,7 +189,17 @@ export default function MainParticipantInfo({ participant, children }: MainParti
           </Tooltip>
         )}
       </div>
-      {(!isVideoEnabled || isVideoSwitchedOff) && (
+      <div
+        className={clsx(classes.avatarContainer, classes.trackSwitchOffContainer, {
```

Comment:
Wondering if `avatarContainer` could be renamed to something more generic since it's now being used to contain something that isn't an avatar.